packs/feat-effects/stance-everstand-stance.json

```diff
@@ -36,6 +36,9 @@
                 "override": {
                     "upgrade": true
                 },
+                "predicate": [
+                    "item:hands-held:2"
+                ],
```

Comment:
This would benefit from getting changed into an item alteration

Reply:
Might as well, on it.